### PR TITLE
chore(website-hub): Generate Hub from repo data

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Replaces https://github.com/cloudquery/cloudquery/pull/1466 pending https://github.com/shuding/nextra/issues/709

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Test locally on your own infrastructure
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: plugins/source/aws/docs/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# AWS Provider
+# AWS Plugin
 
 The CloudQuery AWS provider extracts and transforms your AWS cloud assets configuration into PostgreSQL.
 
 ## Install
 
-```shell
```
Comment (erezrokah) [inline plugins/source/aws/docs/index.md:7]: Nextra supports `bash` and not `shell`

File: plugins/source/aws/docs/index.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ export AWS_PROFILE=myprofile
 
 or, configure your desired profile in the `local_profile` field of your CloudQuery `cloudquery.yml`:
 
-```yml title="cloudquery.yml"
```
Comment (erezrokah) [inline plugins/source/aws/docs/index.md:74]: Nextra supports `yaml` and not `yml`

**Review by dj-stormtrooper — APPROVED** [review]
Great job!

**Review by yevgenypats — APPROVED** [review]
Looks good! The broken links supposed to fail?

**Comment by erezrokah** [comment]
> The broken links supposed to fail?

Looking into it, it's the edit on GitHub feature by Nextra that fails to get the correct link:
image
https://cloudquery-2awwmfwlz-cloudquery.vercel.app/hub/plugins/digitalocean

I only found a way to disable it altogether

**Comment by erezrokah** [comment]
Related to https://github.com/shuding/nextra/issues/662

**Pull request merged**